### ISSUE TITLE
Turn-key persistence transformer

### ISSUE THREAD

**Issue by asayers** [meiersi/blaze-react#21]
It would be easy to write an app-transformer which simply persists the entire app. Might be worth doing as a demo?

**Comment by meiersi** [comment]
Yeah that sounds cool. Ideally a core one based on bytestring or text, and
derived ones using Show/Read or aeson.
Am 12.03.2015 16:25 schrieb "Alex Sayers" <notifications@github.com>:
